Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zcash_script"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
We typically don't update this until we're ready to make a release, which is always done in a separate PR. If you need to refer to this from a different project, use a `[patch.crates-io]` section in that project's `Cargo.toml`